### ISSUE TITLE
ImportError due to breaking change in isort v5

### ISSUE THREAD

**Issue by fphammerle** [WanzenBug/pylint-import-requirements#27]
```python
   File "/src/.venv/lib/python3.8/site-packages/pylint_import_requirements/__init__.py", line 26, in <module>
    from isort import isort
ImportError: cannot import name 'isort' from 'isort' (/src/.venv/lib/python3.8/site-packages/isort/__init__.py)
```

```sh
$ python3 -m venv isort-import-test
$ source isort-import-test/bin/activate
(isort-import-test) $ pip3 install isort
Collecting isort
  Downloading isort-5.4.2-py3-none-any.whl [...] 
Installing collected packages: isort
Successfully installed isort-5.4.2
(isort-import-test) $ python3 -c 'from isort import isort'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'isort' from 'isort' (/tmp/isort-import-test/lib/python3.8/site-packages/isort/__init__.py)
[1](isort-import-test) $ pip3 install 'isort<5'
Collecting isort<5
  Using cached isort-4.3.21-py2.py3-none-any.whl (42 kB)
Installing collected packages: isort
  Attempting uninstall: isort
    Found existing installation: isort 5.4.2
    Uninstalling isort-5.4.2:
      Successfully uninstalled isort-5.4.2
Successfully installed isort-4.3.21
(isort-import-test) $ python3 -c 'from isort import isort' && echo success
success
```

**Comment by fphammerle** [comment]
`isort/isort.py` was removed in https://github.com/PyCQA/isort/commit/a80cc9be4305cc0900b7bf4302b56037e2dfbfdd

`isort/compat.py` including `SortImports` was removed in https://github.com/PyCQA/isort/commit/c7378c3b77a115801200e7a20201075858997a28

**Comment by fphammerle** [comment]
fixed in new release v2.0.5
